addons/account/models/account_move.py

```diff
@@ -776,6 +776,9 @@ def _sequence_year_range_monthly_regex(self):
     highlight_send_button = fields.Boolean(compute='_compute_highlight_send_button')
     is_sale_installed = fields.Boolean(compute='_compute_is_sale_installed')
 
+    # For tracking purposes
+    date_tracking = fields.Json(string="Date Tracking", copy=False, exportable=False)
```

Comment:
Ah yes, this is so much easier than a model that already exists. Let's implement tracking in different ways on all models for each separate field... 👎